demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -169,17 +170,23 @@ def copy_dir_yml(self, dir_path, bundle):
         scan_files, _ = get_yml_paths_in_dir(dir_path, error_msg='')
         content_files = 0
         dir_name = os.path.basename(dir_path)
-        copy_func = self.copy_playbook_yml if dir_name in ['Playbooks', 'TestPlaybooks'] else self.copy_content_yml
         for path in scan_files:
             if len(os.path.basename(path)) >= self.file_name_max_size:
                 self.long_file_names.append(path)
 
             with open(path, 'r') as file_:
                 yml_info = yaml.safe_load(file_)
-
             ver = yml_info.get('fromversion', '0')
             print(f' - processing: {ver} ({path})')
-            copy_func(path, os.path.join(bundle, os.path.basename(path)), yml_info)
+
+            if dir_name in ['Playbooks', 'TestPlaybooks']:
+                self.copy_playbook_yml(path, os.path.join(bundle, os.path.basename(path)))
+            else:
+                ryaml = YAML()
+                ryaml.allow_duplicate_keys = True
+                with io.open(path, mode='r', encoding='utf-8') as file_:
+                    yml_unified = ryaml.load(file_)
```

Comment:
no actual reason to open the file twice (here and on lines 177-178).
read it once using `ruamel` package.

extra: you can also modify the function `Unifier.write_yaml_with_docker` to use only the unified version of the dict so you don't have to pass it twice.
extra2: same goes for the script object, it can be calculated inside the function.

Reply:
Done(I think)